docs/documentation/upgrading/topics/changes/changes-26_4_0.adoc

```diff
@@ -68,6 +68,24 @@ quarkus.datasource.user-store.username=my-username
 
 Notable changes may include internal behavior changes that prevent common misconfigurations, bugs that are fixed, or changes to simplify running {project_name}.
 
+=== Supported Databases Versions
+
+The supported version of each database is now shown in link:{databaseguide_link}[{databaseguide_name}].
+
+It is not a supported configuration if the underlying database specific Hibernate dialect allows the use of a version that differs from those shown.
+
+From 26.3 to 26.4 the support changed as follows:
+
+[cols="1,1,1"]
+|===
+|Database |Minimum support |Dialect minimum
```

Comment:
Do we really want to state dialect minimum? It's not consistent with the guide where we don't state it. It might also be confusing to the users what is "dialect minimum". I'd suggest removing it and limit the table to what we actually support.